enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -462,27 +462,40 @@ compatibility, the Ingress Operator must do one of the following:
 Placement, possible migration, and ownership of the SMCP need to be clarified in
 this enhancement.  This is an open design question.
 
-#### Managing the ServiceMeshMemberRoll
+#### Managing Istiod's Control Plane Scope
```

Comment:
We haven't mentioned Istiod yet, so you might want to choose a different section title.